src/logic/quotes.js

```diff
@@ -0,0 +1,109 @@
+const mathQuotes = [
+  {
+    quote: 'Pure mathematics is, in its way, the poetry of logical ideas.',
+    author: 'Albert Einstein',
+  },
+  {
+    quote:
+      "In mathematics, you don't understand things. You just get used to them.",
+    author: 'John von Neumann',
+  },
+  {
```

Comment:
- [ ] Good work on your Project, however, it i required in the project that you fetch data from an external API [API Ninjas](https://api-ninjas.com/api/quotes). Kindly implement it with the following requirements:

![image](https://user-images.githubusercontent.com/98453979/235209698-3f0cf46d-0885-426c-9741-4d2ac9b31e5d.png)